qa/rpc-tests/test_framework/util.py

```diff
@@ -130,15 +130,15 @@ def waitFor(timeout, fn, onError="timeout in waitFor", sleepAmt=1.0):
     timeout = float(timeout)
     while 1:
         result = fn()
-        if not result is None:
+        if not (result is None or result is False):
```

Comment:
why this change?

Reply:
Its a useability thing.  With this change I can do:
waitFor(10, lambda: node.something() == 50)

Without this change I have to explicitly return None like this:

waitFor(10, lambda: True if node.something() == 50 else None)